roles/openshift_health_checker/library/ocutil.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/python
+"""Interface to OpenShift oc command"""
+
+import os
+import shlex
+import shutil
+import subprocess
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+ADDITIONAL_PATH_LOOKUPS = ['/usr/local/bin', os.path.expanduser('~/bin')]
+
+
+def locate_oc_binary():
```

Comment:
The problem is `lib_openshift` is a s*** load of things, and this "helper" function gets copied to several modules. I think it is not even accessible "as-is".
One way I can think of is to refactor such that there's a module that is responsible for finding out the `oc` binary and then we set the result as a fact. Then, no other module would need this code, instead code would just read the ansible variable.